Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "*"
+gunicorn = "*"
+django-mail-auth = "*"
+python-decouple = "*"
```

Comment:
We should pin the versions here.

Reply:
Isn't that what the `Pipfile.lock` is doing?

Reply:
No, pilock is generated once at first install, whatever the latest version is. Pinning in Pipfile ensure this version is an absolute prerequisite and helps keeping control over it.

Reply:
I pinned the major and middle (if any) versions, and let the minor upgrade